bundles/action/org.openhab.action.pushover/src/main/java/org/openhab/action/pushover/internal/Pushover.java

```diff
@@ -45,7 +46,7 @@
 
     private static final String JSON_API_URL = "https://api.pushover.net/1/messages.json";
     private static final String JSON_CANCEL_API_URL = "https://api.pushover.net/1/receipts/{receipt}/cancel.json";
-    private static final String UTF_8_ENCODING = "UTF-8";
+    private static final String UTF_8_ENCODING = StandardCharsets.UTF_8.name();
```

Comment:
That is okay but not the idea behind my comment. I maybe could have explained it a little bit more in detail.

I think we do not need this constant here. Rather replace it everywhere in this file. One benefit will be that it will be possible to replace it completely in the `IOUtils.toString()` methods by using the `Charset` itself to omit `UnsupportedCharsetException` (or `UnsupportedEncodingException`, see [commons.io API](https://commons.apache.org/proper/commons-io/javadocs/api-2.5/org/apache/commons/io/IOUtils.html#toString(java.io.InputStream,%20java.nio.charset.Charset))).

https://github.com/openhab/openhab1-addons/blob/b4b7a293e85c031693785831685949860829e920/bundles/action/org.openhab.action.pushover/src/main/java/org/openhab/action/pushover/internal/Pushover.java#L510

https://github.com/openhab/openhab1-addons/blob/b4b7a293e85c031693785831685949860829e920/bundles/action/org.openhab.action.pushover/src/main/java/org/openhab/action/pushover/internal/Pushover.java#L605

Reply:
Now I understand what you meant.
Unfortunately, this method isn't available in the current  commons.io API version. 
`The method toString(InputStream, String) in the type IOUtils is not applicable for the arguments (InputStream, Charset)`

Reply:
I see. We are using 2.2 and the expected method is available from 2.3. What a pity.